src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -223,11 +235,45 @@ function createHealthCareRegionListPages(page, drupalPagePath, files) {
     'health_care_region_health_services_page.drupal.liquid',
   );
 
+  // Create the patient and family services page
+  const fsEntityUrl = createEntityUrlObj(drupalPagePath);
+  const fsObj = Object.assign(
+    {
+      careCoordinatorPatientFamilyServices:
+        page.careCoordinatorPatientFamilyServices,
+    },
+    {
+      socialProgramsPatientFamilyServices:
+        page.socialProgramsPatientFamilyServices,
+    },
+    {
+      healthWellnessPatientFamilyServices:
+        page.healthWellnessPatientFamilyServices,
+    },
+    { fieldPatientFamilyServicesIn: page.fieldPatientFamilyServicesIn },
+    { facilitySidebar: sidebar },
+    { entityUrl: fsEntityUrl },
+    { alert: page.alert },
+    { title: page.title },
+  );
```

Comment:
Does this need to be done in an `Object.assign`, or can the object be constructed directly? Ex -

```
{
      careCoordinatorPatientFamilyServices: page.careCoordinatorPatientFamilyServices,
      socialProgramsPatientFamilyServices: page.socialProgramsPatientFamilyServices,
      healthWellnessPatientFamilyServices: page.healthWellnessPatientFamilyServices,
     // ...
}
```

I could be missing something, but otherwise it might make the data structure a little more straightforward.